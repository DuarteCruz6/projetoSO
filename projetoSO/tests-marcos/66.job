WRITE [(y6,38)]
WRITE [(b1,80)(u1,82)(c2,92)(c7,63)(i4,42)]
WRITE [(i3,61)(d4,63)(h0,13)(f7,9)]
WRITE [(j6,54)]
WRITE [(t5,30)(x5,52)(t2,59)]
WRITE [(q9,2)(o9,86)(i9,63)(h5,52)]
WRITE [(l6,58)(e6,100)(g5,77)(c6,40)]
SHOW
WRITE [(c4,28)(h3,18)(m4,93)(d9,64)]
READ [i4]
WRITE [(o1,100)(q1,35)(o6,95)(h9,15)(e4,0)]
WRITE [(t9,37)(b7,51)(d5,56)(z9,42)]
WRITE [(v7,81)(h7,32)(f9,40)]
BACKUP
WAIT 1000
SHOW
WRITE [(s6,3)]
READ [c6]
WRITE [(s2,21)(n7,78)(y9,56)(n5,95)]
WAIT 2000
DELETE [f9,y9,t9]
WRITE [(q8,91)(g8,5)(c9,43)(i7,87)]
WRITE [(b0,60)(z8,19)]
READ [h7]
WRITE [(w3,35)(k5,93)(r5,65)(j5,6)(w0,13)]
WRITE [(m5,57)(e2,33)(z3,10)(q7,62)(k4,59)]
READ [j6]
WRITE [(o0,21)(z6,25)]
READ [j6]
WRITE [(i2,66)(f2,32)(a0,68)(z1,80)(v8,56)]
WRITE [(e8,65)(a8,77)(f1,21)(b9,12)(f6,71)]
DELETE [a8,k5,b7,f1,c9]
READ [b9]
WRITE [(e0,78)(p8,86)(c1,83)(s8,32)(u4,38)]
WRITE [(x8,91)(u2,76)(n6,12)(d1,75)(a5,76)]
WRITE [(f1,32)(s4,25)(j8,59)(x9,84)]
WRITE [(k1,9)(a9,12)(x1,95)(m1,95)]
WRITE [(r3,21)(w9,55)]
WRITE [(m0,80)(g2,50)]
WRITE [(y5,97)(i0,3)(v5,3)(m2,49)(k2,57)]
READ [a0]
READ [i9]
WRITE [(v0,57)(n4,5)(v2,82)(v2,100)]
READ [v0]
WRITE [(h2,25)(p6,57)(k8,23)]
SHOW
DELETE [d1,x5]
READ [x9]
WRITE [(a4,82)(e3,21)]
READ [i3]
WRITE [(k3,37)(j0,44)(u6,70)]
READ [i2]
WRITE [(j7,59)(d6,76)]
DELETE [o9]
READ [h3]
DELETE [h3,s2]
WRITE [(z0,46)]
WRITE [(g1,46)(u9,53)(l3,92)(y9,70)(d2,37)]
SHOW
SHOW
SHOW
WRITE [(i1,98)(t4,85)(a3,73)(o2,83)(f8,15)]
READ [z6]
WRITE [(c5,45)(k5,32)]
WRITE [(w5,2)(r8,84)(m8,30)(e9,12)(n0,60)]
DELETE [c5,n0,i1,z0]
