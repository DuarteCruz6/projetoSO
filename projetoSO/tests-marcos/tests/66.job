WRITE [(y9,28)(r4,59)]
WRITE [(v0,88)(j2,44)(x7,15)(w7,94)(l5,16)]
DELETE [r4,x7,l5,y9,w7]
WRITE [(r2,24)(m3,77)(r9,60)(j5,22)(i9,71)]
WRITE [(k8,57)(q1,21)(b0,50)(e4,70)(x4,67)]
WRITE [(v4,31)(z2,13)(n0,1)(v2,69)]
BACKUP
WRITE [(g2,93)(t1,9)]
READ [m3]
SHOW
WRITE [(l8,66)(e3,30)(s7,0)]
READ [r2]
WRITE [(e2,7)(l5,45)(v3,50)(a0,57)(f9,56)]
WRITE [(n2,100)(u2,37)(y5,86)(y5,78)(r0,85)]
READ [m3]
WRITE [(l4,59)(o8,37)(t5,91)(q8,22)(v5,45)]
WRITE [(h1,79)(c3,72)(n1,88)]
DELETE [g2]
READ [r0]
WRITE [(z7,81)(c2,90)]
WRITE [(r6,0)(l1,63)(n9,82)]
WRITE [(j3,29)(k0,34)(y6,74)(n7,83)]
