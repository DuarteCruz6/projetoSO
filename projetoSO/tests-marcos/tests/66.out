[(m3,77)]
(b0, 50)
(e4, 70)
(g2, 93)
(i9, 71)
(j5, 22)
(j2, 44)
(k8, 57)
(m3, 77)
(n0, 1)
(q1, 21)
(r9, 60)
(r2, 24)
(t1, 9)
(v2, 69)
(v4, 31)
(v0, 88)
(x4, 67)
(z2, 13)
[(r2,24)]
[(m3,77)]
[(r0,85)]
